fabric-fluid-api-v1/src/main/java/net/fabricmc/fabric/api/fluid/v1/render/FluidRenderer.java

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.fluid.v1.render;
+
+import net.fabricmc.fabric.api.client.render.fluid.v1.FluidRenderHandler;
+import net.fabricmc.fabric.api.client.render.fluid.v1.FluidRenderHandlerRegistry;
+import net.fabricmc.fabric.api.event.client.ClientSpriteRegistryCallback;
+import net.fabricmc.fabric.api.resource.ResourceManagerHelper;
+import net.fabricmc.fabric.api.resource.SimpleSynchronousResourceReloadListener;
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.client.texture.Sprite;
+import net.minecraft.fluid.Fluid;
+import net.minecraft.fluid.FluidState;
+import net.minecraft.resource.ResourceManager;
+import net.minecraft.resource.ResourceType;
+import net.minecraft.util.Identifier;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.registry.Registry;
+import net.minecraft.world.BlockRenderView;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.function.Function;
+
+/**
+ * Provides basic fluid rendering.
+ */
+public class FluidRenderer {
```

Comment:
The name of this class is confusing for me. From first glance and without reading the other explanations, I thought this class and its methods were to render a fluid block in the world. To avoid this confusion and a name conflict with vanilla's `FluidRenderer`, I think this class should be renamed to something like `SimpleFluidRenderRegistry`. The methods should be renamed from `render` to `register`, `registerRenderer`, `assignRenderer`, etc.

Reply:
Ok, thanks for noticing, the problem was that I was 17 commits before and there was no SimpleFluidRenderHandler class.
So I've simply removed this duplicate class.

However I think that the texture registration must also be inside the api to avoid potentially forgetfulness and bugs.

```java
ClientSpriteRegistryCallback.event(PlayerScreenHandler.BLOCK_ATLAS_TEXTURE).register((atlasTexture, registry) -> {
  registry.register(...);
  registry.register(...);
});
```

I've spent 15 minutes to understand why my textures wont load... and then I had forgotten to register them... 😅